tikv/pd
# dr-autosync: retry replicate state file

Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?

close #4327

Besides the issue, if a member failed to persist state file, we will not retry.

### What is changed and how it works?

Record members that are successfully replicated. During the tick process (default 1m interval), if it is found that not all members have fully updated their status, a retry will be performed.

### Check List

Tests



- Unit test
- Manual test (add detailed scripts or steps below)
```
- start a 1PD cluster
- wait for the DR_STATE is persisted
- add 2 more PD members
- wait a while, make sure all new members have the DR_STATE file
```

Related changes
- Need to cherry-pick to the release branch

### Release note

```release-note
None
```
